### PR TITLE
✨ Add nav tabs and remove arrow in wnft index page

### PR DESCRIPTION
**Pull request by williamchong**
image

### CODE REVIEW

File: src/pages/writing-nft/index.vue [hunk 1]
```diff
@@ -133,12 +135,50 @@ export default {
       link: [{ rel: 'canonical', href: `${this.$route.path}` }],
     };
   },
+  data() {
+    return {
+      currentTab: this.$route.query.tab || 'featured',
+      trenidngClassIds: [],
```
Comment (AuroraHuang22) [inline src/pages/writing-nft/index.vue:141]:
```suggestion
      trendingClassIds: [],
```

File: src/pages/writing-nft/index.vue [hunk 2]
```diff
@@ -133,12 +135,50 @@ export default {
...
+      latestClassIds: [],
+    };
+  },
   computed: {
     nfts() {
-      return LIKECOIN_NFT_CAMPAIGN_ITEMS;
+      switch (this.currentTab) {
+        case 'trending': {
+          return this.trenidngClassIds;
```
Comment (AuroraHuang22) [inline src/pages/writing-nft/index.vue:149]:
```suggestion
          return this.trendingClassIds;
```

File: src/pages/writing-nft/index.vue [hunk 3]
```diff
@@ -150,6 +190,29 @@ export default {
         // No-op
       }
     }
+    const [trendingRes, latestRes] = await Promise.all([
+      this.$axios.$get(getTopNFTClasses()),
+      this.$axios.$get(getLatestNFTClasses()),
+    ]);
+    [this.trenidngClassIds, this.latestClassIds] = [trendingRes, latestRes].map(
```
Comment (AuroraHuang22) [inline src/pages/writing-nft/index.vue:197]:
```suggestion
    [this.trendingClassIds, this.latestClassIds] = [trendingRes, latestRes].map(
```

**Review by AuroraHuang22 — CHANGES_REQUESTED** [review]

**Review by AuroraHuang22 — APPROVED** [review]

File: src/pages/writing-nft/index.vue [hunk 4]
```diff
@@ -150,6 +190,29 @@ export default {
...
+          .filter(
+            c => c.metadata?.nft_meta_collection_id === 'likerland_writing_nft'
+          )
+          .map(c => c.id)
+          .slice(0, 10)
+    );
+  },
+  methods: {
+    handleTabClick(tab) {
+      this.currentTab = tab;
```
Comment (nwingt) [inline src/pages/writing-nft/index.vue:209]: I think `currentTab` should be a computed prop instead of data, as we can count on `tab` in qs

**Review by nwingt — COMMENTED** [review]
Should add a redirect for `/campaign/writing-nft` to `/writing-nft`?

File: src/pages/writing-nft/index.vue [hunk 5]
```diff
@@ -34,28 +34,34 @@
...
-        </svg>
       </div>
-      <a
-        class="flex items-center justify-center mt-[24px] text-medium-gray hover:text-like-cyan-dark text-[12px] leading-[5/3] underline transition-colors cursor-pointer"
-        href="https://likecoin.github.io/likecoin-nft-dashboard/"
-        target="_blank"
-        rel="noopener"
-      >{{ $t('campaign_nft_view_nft_dashboard') }}</a>
+
+      <nav class="mt-[24px] relative flex-col laptop:flex-row flex items-center justify-center self-stretch gap-[32px]">
```
Comment (nwingt) [inline src/pages/writing-nft/index.vue:39]: I think the navbar is too close to the intro section
```suggestion
      <nav class="mt-[48px] relative flex-col laptop:flex-row flex items-center justify-center self-stretch gap-[32px]">
```

File: src/pages/writing-nft/index.vue [hunk 6]
```diff
@@ -34,28 +34,34 @@
...
+        <slot name="tab-bar-prepend" />
```
Comment (nwingt) [inline src/pages/writing-nft/index.vue:40]: Should remove this slot if not used

**Review by nwingt — APPROVED** [review]

**Pull request merged**